src/rest_framework_dso/pagination.py

```diff
@@ -24,6 +26,8 @@ class DSOHTTPHeaderPageNumberPagination(pagination.PageNumberPagination):
     This can be used for for non-JSON exports (e.g. CSV files).
     """
 
+    _paginator_class = DSOPaginator
```

Comment:
Waarom niet `django_paginator_class` uit de base class overriden? De `DSOPaginator` erft immers ervan. En anders ff commentaar waarom dit niet gedaan is.

Reply:
Ik had het aangepast omdat het niet de _django_ paginator is. Maar nu blijft die property uit de baseclass natuurlijk beschikbaar. Ik zal django_paginator_class weer overriden.